### PR TITLE
BAEL-4886 Flux generate vs create

### PR DESCRIPTION
**Pull request by musibs**

### CODE REVIEW

**Comment by JavierPeris** [comment]
Please @musibs have a look at my comments related to indentation and trailing whitespaces.

File: reactor-core/src/test/java/com/baeldung/reactor/generate/create/CharacterUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+public class CharacterUnitTest {
+    @Test
+    public void whenGeneratingCharacters_thenCharactersAreProduced() {
+        CharacterGenerator characterGenerator = new CharacterGenerator();
+        Flux<Character> characterFlux = characterGenerator.generateCharacters().take(3);
+
+        StepVerifier.create(characterFlux)
+                .expectNext('a', 'b', 'c')
+                .expectComplete()
+                .verify();
```
[the remark this answers is not in the document]
Comment (musibs) [reply reactor-core/src/test/java/com/baeldung/reactor/generate/create/CharacterUnitTest.java]: Corrected.

File: reactor-core/src/test/java/com/baeldung/reactor/generate/create/CharacterUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            .take(3)
+            .collectList()
+            .block();
+        List<Character> sequence2 = characterGenerator.generateCharacters()
+            .take(2)
+            .collectList()
+            .block();
+
+        CharacterCreator characterCreator = new CharacterCreator();
+
```
[the remark this answers is not in the document]
Comment (musibs) [reply reactor-core/src/test/java/com/baeldung/reactor/generate/create/CharacterUnitTest.java]: Removed.

File: reactor-core/src/test/java/com/baeldung/reactor/generate/create/CharacterUnitTest.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    public void whenGeneratingCharacters_thenCharactersAreProduced() {
+        CharacterGenerator characterGenerator = new CharacterGenerator();
+        Flux<Character> characterFlux = characterGenerator.generateCharacters().take(3);
+
+        StepVerifier.create(characterFlux)
+                .expectNext('a', 'b', 'c')
+                .expectComplete()
+                .verify();
+    }
+
```
[the remark this answers is not in the document]
Comment (musibs) [reply reactor-core/src/test/java/com/baeldung/reactor/generate/create/CharacterUnitTest.java]: Removed.

File: reactor-core/src/test/java/com/baeldung/reactor/generate/create/CharacterUnitTest.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    public void whenGeneratingCharacters_thenCharactersAreProduced() {
+        CharacterGenerator characterGenerator = new CharacterGenerator();
+        Flux<Character> characterFlux = characterGenerator.generateCharacters().take(3);
+
+        StepVerifier.create(characterFlux)
+            .expectNext('a', 'b', 'c')
+            .expectComplete()
+            .verify();
+    }
+    @Test
```
Comment (JavierPeris) [inline reactor-core/src/test/java/com/baeldung/reactor/generate/create/CharacterUnitTest.java:24]: Thre´s an empty line missing between methods.

**Pull request merged**